### PR TITLE
Added "(successes/total)" counter to test suite display.

### PR DESCRIPTION
**Pull request by Wcubed**
Adds a counter to the test suites which displays the success / total ratio for that test suite. Updates as each test case finishes.
Unit tests for the feature are included.

Also fixed a typo with "statisics" v.s. "statistics"

image

### CODE REVIEW

**Comment by MikeSchulze** [comment]
Great improvement, thanks. 
I'll look at it soon after I fix the critical update bug.

File: addons/gdUnit3/src/ui/parts/InspectorTreeMainPanel.gd [hunk 1]
```diff
@@ -39,8 +39,11 @@ enum STATE {
 	ERROR,
 	ABORDED
 }
+const META_GDUNIT_NAME := "dgUnit_name"
```
Comment (MikeSchulze) [inline addons/gdUnit3/src/ui/parts/InspectorTreeMainPanel.gd:42]:
```suggestion
const META_GDUNIT_NAME := "gdUnit_name"
```

File: addons/gdUnit3/test/ui/parts/InspectorTreeMainPanelTest.gd [hunk 2]
```diff
@@ -37,8 +37,6 @@ func setup_test_env() -> Array:
 	tsB.add_child(_TestCase.new().configure("test_ba", 10, "/foo/TestSuiteA.gd"))
 	tsB.add_child(_TestCase.new().configure("test_bb", 20, "/foo/TestSuiteA.gd"))
 	tsB.add_child(_TestCase.new().configure("test_bc", 30, "/foo/TestSuiteA.gd"))
-	tsB.add_child(_TestCase.new().configure("test_bd", 40, "/foo/TestSuiteA.gd"))
-	tsB.add_child(_TestCase.new().configure("test_be", 50, "/foo/TestSuiteA.gd"))
```
Comment (MikeSchulze) [inline addons/gdUnit3/test/ui/parts/InspectorTreeMainPanelTest.gd:41]: why you removed this test cases?
A i see you changed to have a different test case count tested on `test_suite_text_shows_amount_of_cases`

File: addons/gdUnit3/test/ui/parts/InspectorTreeMainPanelTest.gd [hunk 3]
```diff
@@ -175,3 +173,50 @@ func test_select_previous_failure() -> void:
...
+	_inspector._on_event(failure_ab)
+	assert_str(suite_a.get_text(0)).is_equal("(1/5) TestSuiteA")
+	
+	var skipped_ac := GdUnitEvent.new().test_after("", "TestSuiteA", "test_ac", {GdUnitEvent.SKIPPED: true})
+	_inspector._on_event(skipped_ac)
+	assert_str(suite_a.get_text(0)).is_equal("(1/5) TestSuiteA")
+	
+	var success_ae := GdUnitEvent.new().test_after("", "TestSuiteA", "test_ae")
+	_inspector._on_event(success_ae)
+	assert_str(suite_a.get_text(0)).is_equal("(2/5) TestSuiteA")
```
Comment (MikeSchulze) [inline addons/gdUnit3/test/ui/parts/InspectorTreeMainPanelTest.gd:222]: very good test coverage +1

**Review by MikeSchulze — CHANGES_REQUESTED** [review]
Only a minor thing the typo, please fix than i can merge this pull request.

Good job!

File: addons/gdUnit3/test/ui/parts/InspectorTreeMainPanelTest.gd (hunk 2 above)
Comment (Wcubed) [reply addons/gdUnit3/test/ui/parts/InspectorTreeMainPanelTest.gd:41]: Yes, I was creating the tests and realized all test suites had 5 cases. Then I saw that these `b` cases weren't actually used in any of the other tests.

File: addons/gdUnit3/test/ui/parts/InspectorTreeMainPanelTest.gd (hunk 3 above)
Comment (Wcubed) [reply addons/gdUnit3/test/ui/parts/InspectorTreeMainPanelTest.gd:222]: Wanted to cover every case. That's how I found out `is_success()` considered a skipped test succesfull :P

**Review by MikeSchulze — APPROVED** [review]

**Pull request merged**